sdk/src/main/java/com/bugsnag/android/ErrorStore.java

```diff
@@ -194,10 +262,14 @@ String getFilename(Object object) {
                 }
             }
         } else {
+            // NDK should always be U + E, as these errors are always fatal
```

Comment:
it says in the description of this PR that this feature only works for JVM errors, so is this code path never decended?

Reply:
That's correct - I can remove this bit.

I originally thought that it would make sense to capture minimal error reports for NDK crashes, but realised that we can't encode the error class at the time of the crash, and as the severity/handled state will always be the same a minimal error report wouldn't be very useful.